### DISCUSSION TITLE
CSS styling not appearing in pdf output

### DISCUSSION THREAD

**Discussion by Hedonical** [quarto-dev/quarto-cli#2046] [Q&A] [answered]
Hi everyone,

I am still getting the hang of Quarto so apologies if this is a basic question. I think I have messed something up in my document setting as my css styling for my document headers is not appearing when I render the pdf.

Here is my raw code for the header:
![Screenshot from 2022-08-22 11-09-46](https://user-images.githubusercontent.com/91704211/185955583-15ddab57-f5eb-4003-9930-0d39cbc25113.png)

It appears properly in the live view:
![Screenshot from 2022-08-22 11-11-08](https://user-images.githubusercontent.com/91704211/185955789-43ef9bf8-ec31-4c0c-906d-913130f69541.png)

but it does not retain the formatting for the pdf output:
![Screenshot from 2022-08-22 11-11-53](https://user-images.githubusercontent.com/91704211/185955938-3ea2f338-2ea3-4e34-a1a3-82a290a0f25e.png)

I suspect I am making a basic mistake with my document setting:

![Screenshot from 2022-08-22 11-13-10](https://user-images.githubusercontent.com/91704211/185956230-f4d2a52a-7828-410a-a6e2-bf407237bfc6.png)

Thanks so much in advance for any comments!

**Comment by cscheid** [comment, marked as answer]
Those are CSS styles, they only work in HTML.

**Reply by Hedonical** [reply]
Thank you!

**Reply by cscheid** [reply]
No problem!

If you want consistent styles, you can either create a format template (which is more elegant but a nontrivial project) or you can use https://quarto.org/docs/authoring/conditional.html to emit different output depending on the format.

**Reply by Hedonical** [reply]
Thank you very much!

**Reply by Hedonical** [reply]
Do you also know in quarto how to only allow css style to apply to the header? I realize that when I make the header of a section blue all of the text for that section also becomes blue.

**Reply by cscheid** [reply]
Can you share a complete example of that happening? Please do so in a new github issue so we can track them more easily.
